frontend/ui/src/actions/conversationsFilter/index.ts

```diff
@@ -110,5 +110,12 @@ const filterToLuceneSyntax = (filter: ConversationFilter): string | null => {
   if (filter.bySources && filter.bySources.length > 0) {
     filterQuery.push('source:(' + filter.bySources.join(' OR ') + ')');
   }
+  if (filter.isStateOpen !== undefined) {
+    if (filter.isStateOpen === true) {
+      filterQuery.push('id:* AND NOT metadata.state:CLOSED');
+    } else {
+      filterQuery.push('metadata.state:CLOSED');
+    }
+  }
```

Comment:
nitpick: I think `if( === true) {...} else if ( !== undefined) {...}` would read a bit better since it's one less indentation.